web/src/app/lists/FlatList.tsx

```diff
@@ -376,9 +405,27 @@ export default function FlatList({
     return <TransitionGroup>{renderTransitionItems()}</TransitionGroup>
   }
 
+  function renderSkeletons(): ReactElement | ReactElement[] {
+    const renderedSkeletons: ReactElement[] = []
+    if (displayLoading) {
+      while (items.length > renderedSkeletons.length) {
+        renderedSkeletons.push(
+          <LoadingItem key={'list_' + renderedSkeletons.length} />,
+        )
+      }
+    }
+    return renderedSkeletons
```

Comment:
I think we can simplify this a bit
```suggestion
    if (!displayLoading) return null

    return items.map((_, idx) => <LoadingItem key={'list_' + idx} />)
```